app/frontend/src/index.scss

```diff
@@ -16,6 +16,7 @@
 @use "pages/Demo/Demo";
 @use "pages/LandingPage/" as *;
 @use "pages/QualifierPage/" as *;
+@use "pages/NotFoundPage/NotFoundPage" as *;
```

Comment:
I _think_ the * is not needed for imports unless it is a composite index scss.

Reply:
I think you are right!